### PR TITLE
[SIG 4098] Containers in Asset select map

### PR DESCRIPTION
**Pull request by kiruys**

### CODE REVIEW

**Comment by boris-arkenaar** [comment]
What's the goal of this PR?

**Comment by kiruys** [comment]
> 

To use asset select map to render the container map. Asset select map is almost a duplicate of Container select map.

File: src/signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/WfsLayer.tsx [hunk 1]
```diff
@@ -93,7 +104,15 @@ const WfsLayer: FunctionComponent<WfsLayerProps> = ({
 
     if (!bbox) return
 
-    const [request, controller] = fetchWithAbort(wfsUrl)
+    if (!wfsUrl) return
```
Comment (boris-arkenaar) [inline src/signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/WfsLayer.tsx:107]: `if (!bbox || !wfsUrl) return`

**Review by boris-arkenaar — APPROVED** [review]
I think I get it :)
Looks good.

File: src/signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/WfsLayer.tsx (hunk 1 above)
Comment (kiruys) [reply src/signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/WfsLayer.tsx:107]: thanks!

**Pull request merged**